Todo-list (events)/hw_7.js

```diff
@@ -0,0 +1,138 @@
+"use strict"
+
+const root = document.getElementById('root');
+root.className = 'container px-5';
+
+const header = document.createElement('header');
+header.className = `d-flex align-items-center justify-content-center 
+    justify-content-md-between border-bottom py-4 mb-4`;
+root.append(header);
+
+const buttonDelete = document.createElement('button');
+buttonDelete.setAttribute('type', 'button');
+buttonDelete.textContent = 'Delete all';
+buttonDelete.className = 'btn btn-primary fs-5 px-4 py-2';
+header.append(buttonDelete);
+
+const form = document.createElement('form');
+form.className = 'd-flex col-10';
+header.append(form);
+
+const enterTask = document.createElement('input');
+enterTask.setAttribute('type', 'text');
+enterTask.setAttribute('placeholder', 'Enter todo ...');
+enterTask.className = 'form-control me-5 fs-5 text-center py-2 text-muted';
+form.append(enterTask);
+
+const buttonAdd = document.createElement('button');
+buttonAdd.setAttribute('type', 'submit');
+buttonAdd.textContent = 'Add';
+buttonAdd.className = 'btn btn-primary fs-5 px-5 py-2';
+form.append(buttonAdd);
+
+const toDoList = document.createElement('ul');
+toDoList.className = 'list-group gap-2 list-unstyled';
+root.append(toDoList);
+
+let toDoItems = JSON.parse(localStorage.getItem('Tasks'));
+
+toDoItems.forEach(renderList);
```

Comment:
вот это не понял зачем
тут хватит просто
```suggestion
renderList();
```